include/simulation_time/Simulation_Time.hpp

```diff
@@ -86,12 +93,23 @@ class Simulation_Time
         return output_interval_seconds;
     }
 
+    /**
+     * @brief Accessor to the the current simulation time
+     * @return current_date_time_epoch
+    */
+
+    time_t get_current_epoc_time()
+    {
+        return current_date_time_epoch;
+    }   
```

Comment:
Please mark all methods that ought to be `const` as such, to ease understanding and guard against mistakes.